discoveries/ecco_and_measures_ssh.csv

```diff
@@ -0,0 +1,602 @@
+date, source, SSH (m)
```

Comment:
I just noticed that the column names have a leading space, which should not happen and is the reason why the chart does not show data.
```suggestion
date,source,SSH (m)
```

Reply:
Fixed - I removed white spaces but I left one in between "SSH" and "(m)". Let's see if that works ...